diff --git a/node_modules/@react-native-firebase/firestore/RNFBFirestore.podspec b/node_modules/@react-native-firebase/firestore/RNFBFirestore.podspec
index 861e763..cfdcb01 100644
--- a/node_modules/@react-native-firebase/firestore/RNFBFirestore.podspec
+++ b/node_modules/@react-native-firebase/firestore/RNFBFirestore.podspec
@@ -41,7 +41,7 @@ Pod::Spec.new do |s|
 
   # required until firestore-ios-sdk-frameworks is updated, otherwise users of that distribution will have compile failures
   # see https://github.com/invertase/firestore-ios-sdk-frameworks/issues/59
-  s.dependency          'nanopb', '>= 2.30908.0', '< 2.30910.0'
+  # s.dependency          'nanopb', '>= 2.30908.0', '< 2.30910.0'
 
   if defined?($RNFirebaseAsStaticFramework)
     Pod::UI.puts "#{s.name}: Using overridden static_framework value of '#{$RNFirebaseAsStaticFramework}'"
diff --git a/node_modules/@react-native-firebase/firestore/android/build/intermediates/aar_metadata/debug/writeDebugAarMetadata/aar-metadata.properties b/node_modules/@react-native-firebase/firestore/android/build/intermediates/aar_metadata/debug/writeDebugAarMetadata/aar-metadata.properties
new file mode 100644
index 0000000..1211b1e
--- /dev/null
+++ b/node_modules/@react-native-firebase/firestore/android/build/intermediates/aar_metadata/debug/writeDebugAarMetadata/aar-metadata.properties
@@ -0,0 +1,6 @@
+aarFormatVersion=1.0
+aarMetadataVersion=1.0
+minCompileSdk=1
+minCompileSdkExtension=0
+minAndroidGradlePluginVersion=1.0.0
+coreLibraryDesugaringEnabled=false
